hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.cpp

```diff
@@ -2838,17 +2671,16 @@ namespace hazelcast {
                     msg.set_partition_id(-1);
 
                     msg.set(uuid);
-                    msg.set(member_uuid);
+                    msg.set(member_u_u_i_d);
```

Comment:
this looks weird to me. maybe we can handle the acronyms specially in the code generator

Reply:
Yes, I should fix this.

Reply:
Fixed https://github.com/hazelcast/hazelcast-client-protocol/blob/3a0bf963ffc07c5065cb94a2f3e35b6ce697bdd2/cpp/__init__.py#L65